instructure/canvas-ios
# Add calendars filter to Planner

refs: MBL-13910
affects: parent
release note: none

Test plan:
* Use the Calendar filter button to filter calendars
* Filter should be unique to children (ie deselecting a course for one
student should not deselect the same course for another student)
* Filter should persist across app launch
* Filter should update the list of planner items
* Calendars filter button should update with the number of selected
calendars

* twilson > student1 > Calendar > February 21
* Should show an assignment for Assignment Overrides
* Tap Calendars > uncheck AO (Assignment Overrides) > Done
* Assignment Overrides assignment should disappear
